media/starboard/BUILD.gn

```diff
@@ -96,10 +96,10 @@ source_set("unit_tests") {
   testonly = true
   sources = []
   if (use_starboard_media) {
-    sources += [
-      "bidirectional_fit_reuse_allocator_test.cc",
-      "starboard_utils_test.cc",
-    ]
+    # TODO(b/379934658): add starboard_renderer_test.
```

Comment:
Instead of removing the sources , my suggestion is to disable the entire target in https://source.corp.google.com/h/github/youtube/cobalt/+/main:media/BUILD.gn;l=205
you would add :


  if (is_cobalt && enable_test_targets) {
     # Add whatever TODO's you want
    deps += ["//media/starboard:unit_tests"]
  }

Reply:
Done.